Solutions/Windows Security Events/Hunting Queries/persistence_create_account.yaml

```diff
@@ -31,8 +30,13 @@ query: |
   | where FileName in~ ("net.exe", "net1.exe")
   | parse kind=regex flags=iU ProcessCommandLine with * "user " CreatedUser " " * "/ad"
   | where not(FileName =~ "net1.exe" and InitiatingProcessFileName =~ "net.exe" and replace("net", "net1", InitiatingProcessCommandLine) =~ ProcessCommandLine)
-  | extend CreatedOnLocalMachine=(ProcessCommandLine !contains "/do")
-  | where ProcessCommandLine contains "/add" or (CreatedOnLocalMachine == 0 and ProcessCommandLine !contains "/domain")
+  | extend CreatedOnLocalMachine=(ProcessCommandLine !has "/do")
+  | where ProcessCommandLine has "/add" or (CreatedOnLocalMachine == 0 and ProcessCommandLine !has "/domain")
   | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), MachineCount=dcount(ComputerName) by CreatedUser, CreatedOnLocalMachine, InitiatingProcessFileName, FileName, ProcessCommandLine, InitiatingProcessCommandLine
   | extend timestamp = StartTimeUtc, AccountCustomEntity = CreatedUser
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: Name
```

Comment:
This needs to be fixed, must have NTDomain or UPNSuffix with Name. https://learn.microsoft.com/en-us/azure/sentinel/entities-reference#user-account

Reply:
@shainw : I tried running this query,  but not able to see on what is getting captured under created user field , 
so didn't understand on how the UPNsuffix and NTdomain can be extracted,

So can you share one value of field over which I will write the extractions?

Reply:
@shainw  , Rest of the changes has been taken care , please review

Reply:
@shainw  , I will raise the separate PR for this Rule, as it requires more work